Makefile

```
@@ -23,6 +23,7 @@ fix-formatting:
 fix-all: fix-codestyle fix-formatting
 
 test-corereadout:
-	./scripts/train_core_readout.py --config-path ../configs --config-name example_train_core_readout_low_res responses_path=https://gin.g-node.org/teulerlab/open-retina/raw/master/responses/eulerlab/rgc_natsim_subset_only_naturalspikes_2024-08-14.h5 +trainer.limit_train_batches=1 trainer.max_epochs=1 +trainer.limit_val_batches=1 +trainer.limit_test_batches=1 natmov_dataloader.batch_size=2
+	python ./scripts/train.py --config-path ../configs --config-name hoefling_2024_core_readout_low_res \
+	data.root_dir=./openretina_cache_folder data.responses_path=https://gin.g-node.org/teulerlab/open-retina/raw/master/responses/eulerlab/rgc_natsim_subset_only_naturalspikes_2024-08-14.h5 \
+	+trainer.limit_train_batches=1 trainer.max_epochs=1 +trainer.limit_val_batches=1 +trainer.limit_test_batches=1 dataloader.batch_size=2
```

```suggestion
        exp_name: test_hoefling_2024_low_res
	+trainer.limit_train_batches=1 trainer.max_epochs=1 +trainer.limit_val_batches=1 +trainer.limit_test_batches=1 dataloader.batch_size=2
```
add a name that makes it obvious that it's just a test. (I forgot this previously)